src/libraries/pretest.proj

```diff
@@ -138,5 +138,20 @@
                              RootAttributes="@(FrameworkListRootAttribute)" />
   </Target>
 
+  <Target Name="ILLinkTrimOOBLibraries"
+          AfterTargets="GenerateRuntimeListFile"
+          DependsOnTargets="SetCommonILLinkArgs"
+          Condition="'$(BuildingNETCoreAppVertical)' == 'true'">
+    <MSBuild Targets="Restore"
+             Projects="$(MSBuildThisFileDirectory)trimming\trimming.csproj"
+             Properties="$(TraversalGlobalProperties);MSBuildRestoreSessionId=$([System.Guid]::NewGuid())" />
```

Comment:
Please avoid this extra restore invocation. We worked really hard in the past months / year to reduce extra invocations. If you put the trimming app under a "src" sub-directory it will be restored automatically. Another option would be to add a P2P in pretest.proj trimming.csproj.

Reply:
I don't think adding a P2P is the right thing to do in this case, specially since it is a Console application project and not a library, but I'll see how I can avoid this manual restore call.

Reply:
That doesn't prevent the P2P. We do have a lot P2Ps to executables i.e. the test runners. Just make sure to set `ReferenceOutputAssembly` to false.